package.json

```diff
@@ -31,6 +17,29 @@
     "node": "^10",
     "yarn": "^1.16.0"
   },
+  "scripts": {
+    "build": "tsc --build",
+    "dev": "concurrently --kill-others \"yarn watch\" \"yarn serve\"",
+    "deploy": "yarn install && yarn demolish && yarn build",
+    "demolish": "rimraf dist",
+    "serve": "nodemon --watch dist --watch config --delay 4 dist/_main.js",
+    "start": "node dist/_main.js",
+    "lint": "yarn lint-tslint && yarn lint-prettier",
+    "lint-tslint": "tslint --config tslint.json --format stylish --project tsconfig.json",
+    "lint-prettier": "prettier --check \"*.{js,json,md,yml}\" \"{.github,@types,config,data,src}/**/*.{js,json,md,ts,yml}\"",
+    "format": "yarn format-prettier",
+    "format-tslint": "tslint --config tslint.json --format stylish --project tsconfig.json --fix",
+    "format-prettier": "yarn lint-prettier --write",
+    "test": "jest --coverage",
+    "coverage": "jest --collectCoverageFrom=src/**.ts --coverage src",
+    "watch": "tsc --watch"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn format",
```

Comment:
This will run the fix command pre-commit but won't add the files to the commit

Reply:
I changed it to lint on commit and build on push.